Sources/MapboxMaps/Foundation/DisplayLinkProtocol.swift

```diff
@@ -15,4 +15,31 @@ internal protocol DisplayLinkProtocol: AnyObject {
     func invalidate()
 }
 
+internal final class EmptyDisplayLink: DisplayLinkProtocol {
+    let timestamp: CFTimeInterval = .nan
+    let duration: CFTimeInterval = .nan
+    var isPaused: Bool = false
+    var preferredFramesPerSecond: Int = 0
+#if swift(>=5.5)
```

Comment:
It occurs to me that this is somewhat problematic because it means these APIs won't be available to anyone using the direct download installation method. We have to build those XCFrameworks with our min supported xcode to ensure bitcode compatibility. I don't think we should change anything about this, but it's something to be aware of in case anyone runs into trouble.

cc: @ZiZasaurus